osf/migrations/0074_parse_citation_styles.py

```diff
@@ -75,5 +75,4 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.RunPython(parse_citation_styles, revert),
```

Comment:
Is there a reason to keep `def parse_citation_styles(state, schema):` and `def get_style_files(path):`?

Reply:
IIRC we are going to just bulk delete all these migration files later anyway.

Reply:
Yes we are. However, for the PR's integrity, I would suggest removing these functions since they are moved to the migration command.

Reply:
I realized we don't remove them for other PRs. Thus I took my suggestion back. Consider resolved.